azurelinuxagent/common/osutil/gaia.py

```diff
@@ -150,11 +150,11 @@ def deploy_ssh_pubkey(self, username, pubkey):
     def eject_dvd(self, chk_err=True):
         logger.warn('eject is not supported on GAiA')
 
-    def mount(self, device, mount_point, option="", chk_err=True):
-        logger.info('mount {0} {1} {2}', device, mount_point, option)
-        if 'udf,iso9660' in option:
+    def mount(self, device, mount_point, option=[], chk_err=True):
+        logger.info('mount {0} {1} {2}', device, mount_point, " ".join(option))
+        if filter(option, lambda opt: 'udf,iso9660' in opt):
```

Comment:
the original condition `'udf,iso9660' in option` should also work with the array of strings

Reply:
I wanted to preserve the original functionality. The original condition would only work in the case that string `'udf,iso9660'` is exactly in the array, whereas the current implementation is fuzzier (like the original condition would've been with the original `option`).

Reply:
ok, let's leave as is to preserve the previous behavior